inginious/frontend/plugins/task_editorial/pages/templates/task_editorial_preview.html

```diff
@@ -0,0 +1,69 @@
+$def with (course, task, all_languages)
```

Comment:
Here you can add more parameters to pass the solution code, tutorial and additional information to render.